src/main/java/net/dv8tion/jda/api/events/automoderation/AutoModerationRuleCreateEvent.java

```diff
@@ -0,0 +1,92 @@
+/*
+ * Copyright 2015 Austin Keener, Michael Ritter, Florian Spieß, and the JDA contributors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package net.dv8tion.jda.api.events.automoderation;
+
+import net.dv8tion.jda.api.JDA;
+import net.dv8tion.jda.api.entities.AutoModerationRule;
+import net.dv8tion.jda.api.entities.TextChannel;
+import net.dv8tion.jda.api.entities.TriggerType;
+import net.dv8tion.jda.api.entities.User;
+import org.jetbrains.annotations.NotNull;
+
+/**
+ * Indicates that a {@link net.dv8tion.jda.api.entities.AutoModerationRule AutoModerationRule} was created.
+ *
+ * <p>Can be used to retrieve the created AutoModerationRule.
+ */
+public class AutoModerationRuleCreateEvent extends GenericAutoModerationEvent {
+
+    private final long ruleId;
+    private final TriggerType triggerType;
+    private final User user;
+    private final TextChannel channel;
+    private final Long messageId;
+    private final Long alertSystemMessageId;
+    private final String content;
+    private final String matchedKeyword;
+    private final String matchedContent;
```

Comment:
`GenericAutoModerationEvent` already contains the `AutoModerationRule` affected by this event, why re-declare all the fields used in `AutoModerationRule`?